model/oauth/client.go

```diff
@@ -814,4 +814,17 @@ func BuildLinkedAppScope(slug string) string {
 	return fmt.Sprintf("@%s/%s", consts.Apps, slug)
 }
 
+func CheckOAuthClientsLimitReached(i *instance.Instance, limit int) (reached, exceeded bool) {
+	clients, _, err := GetConnectedUserClients(i, 100, "")
```

Comment:
s/100/limit+1/

Reply:
I did that in the first place but we figured out the clients count wouldn't be accurate if there are more clients than `limit + 1`.
I used `100` because we shouldn't have a limit this high ever.